README.es.md

```diff
@@ -29,10 +29,10 @@ Si aún no puedes encontrarlo, comprueba si el paquete ya [incluye](https://www.
 Esto es provisto usualmente en el campo `"types"` o `"typings"` en el `package.json`,
 o solo busca por cualquier archivo ".d.ts" en el paquete e inclúyelo manualmente con un `/// <reference path="" />`.
 
-#### Versiones más viejas de TypeScript (3.7 y anteriores)
+#### Versiones más viejas de TypeScript (3.8 y anteriores)
```

Comment:
looking at https://github.com/microsoft/DefinitelyTyped-tools/blob/305cdef85c9ea5e490cf39140aa319638d4166bd/packages/typescript-versions/src/index.ts#L49, this should now be 3.9 because UnsupportedTypescriptVersion now includes 3.9